packages/website/docs/getAlgoliaFacetHits.md

```diff
@@ -29,23 +59,33 @@ getAlgoliaFacetHits({
 });
 ```
 
-## Params
+## Parameters
 
 ### `searchClient`
 
 > `SearchClient` | required
 
+The initialized Algolia search client.
+
 ### `queries`
 
+> `FacetQuery[]` | required
+
+The queries to search for, with the following parameters:
+
 #### `indexName`
 
 > `string` | required
 
+The index name.
+
 #### `params`
 
-> [`SearchParameters`](https://www.algolia.com/doc/api-reference/search-api-parameters/) & [`Request Options`](https://www.algolia.com/doc/api-client/getting-started/request-options/) | required
+> [`SearchForFacetValuesQueryParams` & `SearchOptions`](https://www.algolia.com/doc/api-reference/api-methods/search-for-facet-values/#parameters)
+
+Algolia search for facet values parameters.
 
-Default search parameters:
+These are the default parameters. You can leave them as is and specify other parameters, or override them.
```

Comment:
If they override `highlightPreTag` and `highlightPostTag`, they won't be able to use our highlighting utils (e.g., `highlightHit`). It might be worth mentioning.

Reply:
Addressed in 1b92ec13. I'll do it in the other preset functions where you can pass search parameters once all is merged.